sys/clif/clif.c

```diff
@@ -299,10 +302,11 @@ ssize_t clif_get_attr(const char *input, size_t input_len, clif_attr_t *attr)
     if (scan_value) {
         /* iterate over value */
         while (pos < end) {
-            if (quoted) {
+            if (quotes == 1) {
                 if (*pos == '"' && *(pos - 1) != '\\') {
                     /* found unescaped quote */
                     attr->value_len = pos - attr->value;
+                    quotes++;
```

Comment:
Why not decrement here to balance the quotes?
```suggestion
                    quotes--;
```

Reply:
I don't understand, here unescaped quotes were found (so the closing quotes), the count should increase.

Reply:
My proposal was, to not use `quotes` as a counter of quotes, but as a "stack", with incrementing being push and decrementing being pop. The string has parsed sane when `quotes` is 0.